db/schemaGenerator/data_generators.js

```diff
@@ -0,0 +1,88 @@
+const words = require('./words');
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+var randomNum = (min = 0, max = min) => {
+    return min + Math.floor(Math.random() * ((max + 1) - min));
+}
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+var sentenceGenerator = (minNumOfWords, maxNumOfWords, possiblyComma) => {
+    var tempMem_chosenIndexes = [];
+    var sentence = '';
+    var numOfWords = randomNum(minNumOfWords, maxNumOfWords);
+    var placement;
+    if (possiblyComma) {
```

Comment:
Thank you! I'll have to check out the API generator though, probably more time-efficient.

And yes, the "possiblyComma" logic is in case I want the sentence to have a comma.
So I invoke it like this:
sentenceGenerator(5, 7 , 1);

1 - yes, include comma
0 - no, don't include comma

But might be better to use booleans instead instead of 0 and 1. Not sure.